app/views/games/index.html.erb

```diff
@@ -1 +1,165 @@
-<h1>StoicChess</h1>
\ No newline at end of file
+<br />
+<section id="register">
+  <div class="col-sm-5 col-sm-offset-1">
+    <%= image_tag 'chess.jpg', class: 'img-responsive feature_pic' %>
+    <br />
+  </div>
+  <div class="col-sm-3 col-sm-offset-1">
+    <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+        <div class="form-inputs">
+          <br />
```

Comment:
Yup definitely, but in this instance it's fine. Nuances are quite ridiculous, but for example we can see that Bootstrap 4 will contain basically the same code: http://v4-alpha.getbootstrap.com/components/utilities/#spacing